src/main/java/bio/terra/app/configuration/DataRepoJdbcConfiguration.java

```diff
@@ -12,6 +13,7 @@
 @EnableConfigurationProperties
 @EnableTransactionManagement
 @ConfigurationProperties(prefix = "db.datarepo")
+@ConditionalOnMissingClass("bio.terra.app.configuration.EmbeddedDataRepoTestConfiguration")
```

Comment:
I'm not sure I understand what this is doing, but it looks like it's trying to conditionally define a component based on another value being defined. I thought that spring profiles are usually how this is done; is it possible to use those instead? It might be clearer than having this code need to know about another class definition.